### PR TITLE
Adds a new strategy to color x-rays by intensity.

### PR DESCRIPTION
**Pull request by SirVer**

### CODE REVIEW

**Comment by SirVer** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user SirVer. Merge queue now has a length of 1.

File: xray/src/bin/build_xray_tile.rs [hunk 1]
```diff
@@ -31,12 +32,28 @@ fn parse_arguments() -> clap::ArgMatches<'static> {
             clap::Arg::with_name("resolution")
                 .help("Size of 1px in meters.")
                 .long("resolution")
-                .default_value("0.01"),
+                .default_value("0.05"),
```
Comment (thomasschiwietz) [inline xray/src/bin/build_xray_tile.rs]: changing the default value is unrelated to this PR, right?

Comment (SirVer) [reply xray/src/bin/build_xray_tile.rs]: Sorry, I missed this comment. yes it is, and I snuck a CI unbreaking change into this as well. I am sorry and will repent :(

**Pull request merged**